SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/DormancyTests/RefreshActorDormancyTestActor.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "RefreshActorDormancyTestActor.h"
+
+#include "Engine/Classes/Components/StaticMeshComponent.h"
+#include "Engine/Classes/Materials/Material.h"
+#include "Net/UnrealNetwork.h"
+
+ARefreshActorDormancyTestActor::ARefreshActorDormancyTestActor()
+{
+	NetDormancy = DORM_Initial;
+}
+
+bool ARefreshActorDormancyTestActor::ReplicateSubobjects(UActorChannel* Channel, FOutBunch* Bunch, FReplicationFlags* RepFlags)
```

Comment:
nit: recommend changing this to PreReplication as it's slightly simpler and more obvious your intention